### DISCUSSION TITLE
License Conversion or Refund Request

### DISCUSSION THREAD

**Discussion by owallacevitor** [aclap-dev/video-downloadhelper#1054] [Help and Q&A]
### Your question

Dear Support Team,

I purchased the VideoDownloader Helper license for the Chrome version on July 11th. However, I recently discovered that the Chrome version does not allow me to download YouTube videos. I would like to know if there is any way to convert my Chrome license to use it on Mozilla Firefox or if there is still a possibility of a refund so that I can purchase the Mozilla Firefox version.

I look forward to your response and thank you in advance for your attention.

**Comment by GrampaWildWilly** [comment]
I don't work for VDH.  I'm just another user.  So I can't help you with your license.

This seems like a perfect duplicate of #1051.  It is not necessary to wait for a refund to buy the other license.  Plenty of people actually own both licenses.  You should think of purchasing a license for VDH for Firefox as one activity.  You should think of getting your refund for your VDH for Chrome license as an entirely unrelated activity.  The rule is that if it's been less than 1 month since you bought a license, @jcfrog Jérôme will gladly refund your purchase, no questions asked.  You've made your refund request here so you just have to wait for Jérôme to respond.  Remember, he's in France so take the time zone difference into account.  But you could buy your license for VDH for Firefox right now, no need to wait.

**Comment by paulrouget** [comment]
https://github.com/aclap-dev/video-downloadhelper/wiki/I-need-help#license-problem
